src/components/app.js

```diff
@@ -4,14 +4,15 @@ import { BrowserRouter } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.css';
 import 'react-toastify/dist/ReactToastify.css';
 import Routes from './Routes';
+import Footer from './common/footer/Footer';
 
 const App = () => (
   <div>
     <BrowserRouter>
       <ToastContainer />
       <Routes />
     </BrowserRouter>
-    ,
+    <Footer />
```

Comment:
Good work @salviosage, if you will, please confirm if there's a Footer component in common.

Reply:
yeah I created it, @Anguandia